modules/developer-cli-odo-ref-storage.adoc

```diff
@@ -0,0 +1,124 @@
+:_content-type: REFERENCE
+[id="odo-storage_{context}"]
+= odo storage
+
+
+`odo` lets users manage storage volumes that are attached to the components. A storage volume can be either an ephemeral volume using an `emptyDir` Kubernetes volume, or a link:https://kubernetes.io/docs/concepts/storage/volumes/#persistentvolumeclaim[Persistent Volume Claim] (PVC). A PVC allows users to claim a persistent volume (such as a GCE PersistentDisk or an iSCSI volume) without understanding the details of the particular cloud environment. The persistent storage volume can be used to persist data across restarts and rebuilds of the component.
+
+== Adding a storage volume
+
+To add a storage volume to the cluster, run the command:
+
+[source,terminal]
+----
+$ odo storage create
+----
+
+.Example output:
+[source,terminal]
+----
+$ odo storage create store --path /data --size 1Gi
+✓  Added storage store to nodejs-project-ufyy
+
+$ odo storage create tempdir --path /tmp --size 2Gi --ephemeral
+✓  Added storage tempdir to nodejs-project-ufyy
+
+Please use `odo push` command to make the storage accessible to the component
+----
+
+
+In the above example, the first storage volume has been mounted to the `/data` path and has a size of `1Gi`,
+and the second volume has been mounted to `/tmp` and is ephemeral.
```

Comment:
Question: Should tL30 and L31 not be in one line? Please check and do the needful.

Reply:
I think this is an artefact of the conversion from markdown to adoc and doesn't affect the output - but I will remove the newline.